scripts/globals/items/sprinters_shoes.lua

```diff
@@ -0,0 +1,26 @@
+-----------------------------------------
+-- ID: 15754
+-- Item: Sprinter's Shoes
+-- Item Effect: Quickening for 60 minutes
+-----------------------------------------
+
+require("scripts/globals/settings");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    return 0;
+end;
+
+
+-----------------------------------------
+-- OnItemUse
+-----------------------------------------
+
+function onItemUse(target)
+    target:delStatusEffect(EFFECT_QUICKENING);
+    target:addStatusEffect(EFFECT_QUICKENING, 10, 0, 3600);
+    target:messageBasic(266,0, EFFECT_QUICKENING);
```

Comment:
adding the effect already should trigger this message. Even if it didn't,  we should
```lua
    GAINS_EFFECT           = 266, -- <target> gains the effect of <status>.
```
to the msg global instead off using "266" directly.

Reply:
However, I cannot get any of these items to trigger the message without the messageBasic line
https://github.com/DarkstarProject/darkstar/blob/2ca782d2c7cbdd3582827ea995c294d99595f176/scripts/globals/items/hermes_quencher.lua#L31
Maybe I'm doing something wrong? It triggers a message when the effect wears off though.

Reply:
I dunno seems to be missing globally, was sure it worked at one point..We even have a flag to suppress the msg in place..

Pull/rebase onto latest and then try this.

[sprinters_shoes.zip](https://github.com/DarkstarProject/darkstar/files/1420084/sprinters_shoes.zip)

Reply:
Yes, that works now as I see you've added the GAINS_EFFECT msg.